src/app/UI/widgets/card/card.component.html

```diff
@@ -1 +1,32 @@
-<p>card works!</p>
+<!--span>{{ data.cloth }} {{ data.brand }} {{ data.description }} {{ data.color }} {{ data.size }} {{ data.price }} {{ data.status }} {{ data.photo}}</span-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Card Clothes</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&display=swap">
+  <link rel="stylesheet" href="card.component.scss">
+</head>
+<body>
```

Comment:
```suggestion
<div class="container">...</div>
```
This is totally wrong, if you inspect your app, you can see that you have now two `html` tags, two `head` and two `body`. Please get rid off them.